guides/advanced_guides/README.md

```diff
@@ -9,7 +9,7 @@ In the advanced guides you will find how to tune your search API and customize i
 - [Stop words](stop_words.md)
 - [Distinct](distinct.md)
 
-You can also read the following articles to understand how MeiliSearch works under the hood and better understand the algorithms used:
+To learn how MeiliSearch works under the hood and better understand the algorithms behind it, you can read the following articles:
```

Comment:
My mistake when I used article. Should we even call them `advanced guides`?
```suggestion
To learn how MeiliSearch works under the hood and better understand the algorithms behind it, you can read the following guides:
```

Reply:
Yes advanced guides would be fine

Reply:
I updated my suggestion